templates/taxi/car_detail.html

```diff
@@ -7,6 +7,8 @@ <h1>Drivers</h1>
   <ul>
   {% for driver in car.drivers.all %}
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+    <p><a href="{% url 'taxi:car-update' pk=car.id %}">Update car</a></p>
+    <p><a href="{% url 'taxi:car-delete' pk=car.id %}">Delete car</a></p>
```

Comment:
The 'Update car' and 'Delete car' links are placed inside the loop iterating over drivers. This causes the links to be repeated for each driver. Consider moving these links outside the loop to display them only once.